cmd/zoekt-dynamic-indexserver/main_test.go

```diff
@@ -0,0 +1,96 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"log"
+	"net/http"
+	"os"
+	"os/exec"
+	"reflect"
+	"strings"
+	"testing"
+	"time"
+)
+
+var (
+	CmdTimeout = 100 * time.Millisecond
+)
+
+func captureOutput(f func()) string {
+	var buf bytes.Buffer
+	log.SetOutput(&buf)
+	f()
+	log.SetOutput(os.Stderr)
+	return buf.String()
+}
+
+func TestLoggedRun(t *testing.T) {
+	ctx, cancel := context.WithTimeout(context.Background(), CmdTimeout)
+	defer cancel()
+
+	cmd := exec.CommandContext(ctx, "echo", "-n", "1")
+
+	stdout := captureOutput(func() {
+		out, err := loggedRun(cmd)
```

Comment:
err is the idiomatic name for something with the type `error`. Rather call this var stderr.